website/docs/docs/authentication/legacy.md

```diff
@@ -0,0 +1,490 @@
+1. Choose Create.
+
+![Okta create app](/saml/okta-saml-create-app.png)
+
+#### 2. Configure SAML integration for your Okta app
+
+1. On the Create SAML Integration page, under General Settings, enter a name for your app.
+1. (Optional) Upload a logo and choose visibility settings for your app.
+1. Choose Next.
+1. Under GENERAL, for Single sign on URL, enter `https://pact-saas-prod-1.auth.ap-southeast-2.amazoncognito.com/saml2/idpresponse`.
+1. For Audience URI (SP Entity ID), enter urn: `urn:amazon:cognito:sp:ap-southeast-2_x0L1olP0D`
+1. Under ATTRIBUTE STATEMENTS (OPTIONAL), add 3 statements with the following information:
+   1. For Name, enter the SAML attribute name `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`.
+   1. For Value, enter `user.firstName`.
+   1. For Name, enter the SAML attribute name `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`.
+   1. For Value, enter `user.lastName`.
+   1. For Name, enter the SAML attribute name `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`.
+   1. For Value, enter `user.email`.
+1. Choose Next.
+1. Choose a feedback response for Okta Support.
+1. Choose Finish.
+
+![Okta create app](/saml/okta-saml-settings-screen.png)
+
+#### 3. Assign a user to your Okta application
+
+1. On the Assignments tab for your Okta app, for Assign, choose Assign to People.
+1. Next to the user you want to assign, choose Assign.
+1. Note: If this is a new account, the only option is to choose yourself (the admin) as the user.
+1. Choose Save and Go Back. Your user is assigned.
+1. Choose Done.
+
+![Assign user screen in Okta](/saml/okta-saml-assign-user-screen.png)
+![Assign a user in Okta](/saml/okta-saml-assign-user.png)
+
+#### 4. Get the IdP metadata for your Okta application
+
+1. On the Sign On tab for your Okta app, find the Identity Provider metadata hyperlink. Right-click the hyperlink and copy the URL.
+1. For more information, see Configure SAML in your app in the Set up a SAML application in Okta guide on the Okta Developer website.
+
+#### 5. Contact PactFlow to enable your IdP
+
+Send your unique metadata URL to us [here](https://support.smartbear.com/pactflow/message/).
+
+### Azure Active Directory
+
+See https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/configure-federated-single-sign-on-non-gallery-applications for more information.
+
+1. Create a non-gallery application
+
+![Create a non-gallery app](/saml/azure-add-non-gallery-application.png)
+
+2. Follow the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-non-gallery-app) for creating a non-gallery application.
+
+- Choose `Non-gallery application` at the `Add your own app` screen.
+- Set the name to `pactflow` when prompted.
+
+![Create a non-gallery app](/saml/azure-add-pactflow.png)
+
+2. When the application has been created, assign the users that should be allowed to login to PactFlow.
```

Comment:
After creating the application, assign the users that should be allowed to login to PactFlow.